recipes/vmtk/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set name = "vmtk" %}
+{% set version = "1.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/vmtk/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: ec52357bf6dfd975eaae592b6b70db67c7767a01278386373588a29cf52997bb
+
+build:
+  number: 0
+  features:
+    - vc14  # [win and (py35 or py36)]
```

Comment:
I am not familiar with features. Can someone else from @conda-forge/staged-recipes take a look here?

Reply:
This should be removed

Reply:
Removed these lines in the latest commit.